assets/src/application/ui/input/DatePicker/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import classNames from 'classnames';
+import { DatePicker as DatepickerAdapter, DatePickerProps } from '@infraUI/inputs';
+
+const DatePicker: React.FC<DatePickerProps> = ({ className, ...props }) => {
+	const htmlClass = classNames(className, 'ee-date-picker', 'ee-calendar-datetime-picker', 'ee-input-base-wrapper');
```

Comment:
```suggestion
	const className = classNames(props.className, 'ee-date-picker', 'ee-calendar-datetime-picker', 'ee-input-base-wrapper');
```

Reply:
was done that way to prevent the `className` prop from being applied to the adapter:

```js
		<div className={htmlClass}>
			<DatepickerAdapter required {...props} />
		</div>
	);
```

which could cause css problems

Reply:
understood, thx for explanation

Reply:
this will cause re-renders 
```
  delete props.className;
```

***
let's leave as is, will deal later with consistencies, not a biggie

Reply:
ok thnx